src/AspNetCoreController/Example/ExecutionResultActionResult.cs

```diff
@@ -18,10 +17,10 @@ public ExecutionResultActionResult(ExecutionResult executionResult)
 
         public async Task ExecuteResultAsync(ActionContext context)
         {
-            var writer = context.HttpContext.RequestServices.GetRequiredService<IDocumentWriter>();
+            var writer = context.HttpContext.RequestServices.GetRequiredService<IGraphQLSerializer>();
```

Comment:
```suggestion
            var serializer = context.HttpContext.RequestServices.GetRequiredService<IGraphQLSerializer>();
```

Reply:
@Shane32 I began to appear a little time to support, but it is still much less than before.